backend/api/routes.py

```diff
@@ -1,88 +1,94 @@
-from flask import Blueprint
+from flask import Blueprint, request
 from backend.database.models import *
+from backend.database.Search import Search
 
 api = Blueprint('api', __name__)
 
 
-@api.route('/buildings')
-def get_all_buildings():
-    try:
-        return Buildings.all()
+@api.route('/points')
+def get_points():
+    if request.args.get('search'):
+        Points.where(Search.points(request.args.get('search')))
```

Comment:
See comment in search class below about converting bad search param to 400 error - maybe we should catch any exceptions caused by bad params here and throw `400 bad request` (presumably thru flask)

Reply:
See comment below